Sources/Collections/Observable2DArray.swift

```diff
@@ -27,17 +27,17 @@ import Diff
 
 public enum Observable2DArrayChange {
   case reset
-
+  
```

Comment:
Would you mind enabling these two options https://i.stack.imgur.com/KjGnz.png so we don't get the unnecessary changes?